.github/workflows/test-only.yml

```diff
@@ -201,6 +201,30 @@ jobs:
           # @todo - scan the container log to see the right "graceful shutdown" text exists           
           docker rm sig-test
 
+      - name: Test changedetection.io wsgi fail case
+        run: |
+
+          echo "wsgi fail case: Address already in use"
+          docker run --name sig-test test-changedetectionio bash -c '
+          # just occupy external port 5000
+          python -m http.server 5000 --bind 0.0.0.0 &
+          # just a second.
+          sleep 1
+          # port conflicts and shutdown.
+          PORT=5000 python changedetection.py -d /datastore' || true &
+          sleep 7
+          docker ps -a
```

Comment:
sleep is a little longer. the reason will be appeared in `docker ps -a`

Reply:
https://github.com/Constantin1489/changedetection.io/actions/runs/7917779129/job/21614650342#step:10:57